core/Tracker/Response.php

```diff
@@ -38,7 +40,7 @@ public function init(Tracker $tracker)
 
     public function getOutput()
     {
-        $this->outputAccessControlHeaders();
+        CORSHandler::outputAccessControlHeaders();
```

Comment:
Actually I think that method call isn't needed anymore. The `piwik.php` calls the handle method before anything is done, which should already send the required headers with the same values. will simply remove it...